pom.xml

```diff
@@ -248,6 +248,35 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.ec4j.maven</groupId>
+					<artifactId>editorconfig-maven-plugin</artifactId>
```

Comment:
I still like `indent_size=2` esepcially for `XML` files. So adapt `.editorconfig` to either set `indent_size=2` for `[*]` or add `xml` to `[*.{fix,gradle,json,md,mwe2,sh,vim,xtext,yml}]
`.

Reply:
Did that